reactjs/src/components/HorizontalCard.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import Badge from "./Badge";
+import { badges as allBadges, Moderator } from "../data/data";
+
+const HorizontalCard: React.FC<Moderator> = ({
+  imageSrc,
+  badges,
+  description,
+  github,
+  website,
+  name,
+}): JSX.Element => {
+  return (
+    <div className="bg-gray-0 h-32 md:h-48 lg:h-48">
+      <div className="flex flex-inline">
+        {imageSrc ? (
+          <div className="h-32 w-32 lg:h-48 lg:w-48 md:h-48 md:w-48">
+            <img className="rounded-full" src={imageSrc} alt={name} />
+          </div>
+        ) : null}
+        <div className="lg:w-2/3 md:w-2/3 sm:w-2/3 px-5 pt-5 sm:pt-2 lg:pt-5 rounded-r-lg">
+          {badges.map((badge, idx) =>
+            badge.toLocaleLowerCase() ===
+            allBadges.community.title
+              .trim()
+              .split(" ")
+              .join("")
+              .toLocaleLowerCase()
+              .toLocaleLowerCase() ? (
+              <Badge key={idx} classes={allBadges.community.classes} title={badge} />
+            ) : badge.toLocaleLowerCase() ===
+              allBadges.opensource.title
+                .trim()
+                .split(" ")
+                .join("")
+                .toLocaleLowerCase()
+                .toLocaleLowerCase() ? (
```

Comment:
is this supposed to be duplicated?

Reply:
No, This should not be duplicated.

duplicaltion will be removed in next commit.
 
Thanks for feedback